.travis.yml

```diff
@@ -31,11 +31,13 @@ rvm:
   - 2.5
   - 2.6
   - 2.7
+  - 3.0
+  - 3.1
 
 script:
   # Please add to Rakefile instead of this section.
   - bundle exec rake
-  - bundle exec rubocop
+  - if [ $TRAVIS_RUBY_VERSION != "3.1" ]; then bundle exec rubocop; fi
```

Comment:
The version of rubocop installed is too old so running it against Ruby 3.1 raises an error: `wrong number of arguments (given 2, expected 1)`

In order to fix it we have to bump rubocop to `~> 1.22`, but for now I figured out it could be better to just avoid running `rubocop` against Ruby 3.1 so we allow the TravisCI to test against Ruby 3.1 and pass.

Reply:
Bumping `rubocop` to `v1` will make it so that `bundle install` fails for the older Rubies (`< 2.4`). We could get around this by pinning the specifics versions that are allowed for each version in the Gemfile but will also need to have different `rubocop.yml` files for each version given that a lot of cops changed their name or were moved to a different category 😥

Reply:
Another option could be to remove the development dependency of `rubocop` for older Rubies and just run `rubocop` in the CI for Ruby `3.1`, but I felt like it was more of an effort for a separate PR.